ranger2/aws/ranger_asg_pvt_subnet/setup_file_chk.sh

```diff
@@ -5,7 +5,7 @@ sleep 120
 
 loop=1
 while [ ! -f /tmp/setup-done ]; do
-    if [ $loop -gt 20 ]; then
+    if [ $loop -gt 50 ]; then
```

Comment:
maybe make this configurable too. and have some debug messages around this as well.